streaming_form_data/_parser.pyx

```diff
@@ -275,27 +276,24 @@ cdef class _Parser:
                 if byte != c_lf:
                     self.mark_error()
                     return ErrorGroup.PartHeaders + 1
+                
+                m = Parser(policy=HTTP).parsestr(chunk[buffer_start: idx + 1].decode('utf-8'))
```

Comment:
Let's make this variable name a bit more descriptive. Perhaps something like `message`?

Reply:
Also done in 188232e